here-naksha-app-service/src/main/java/com/here/naksha/app/service/http/tasks/StorageApiTask.java

```diff
@@ -149,4 +155,22 @@ private static String mismatchMsg(String storageIdFromPath, Storage storageFromB
     return "Mismatch between storage ids. Path storage id: %s, body storage id: %s"
         .formatted(storageIdFromPath, storageFromBody.getId());
   }
+
+  private static final String JSON_KEY_PASSWORD = "password";
```

Comment:
According to most code conventions I came across (including Google's and Oracle's for Java) class variables (aka: static ones) should be placed on top of the class body. Then instance variables, then constructors, then method. You can also try to group them by visibility but this is not that strict.

Mixing static variables like that between function declarations looks a bit odd ;)
To have some reference:
- https://www.oracle.com/java/technologies/javase/codeconventions-contents.html

summing it up briefly - in order:
- Class (static) variables
- Instance variables
- Constructor
- Methods